File: src/main/java/NetworkUtil.java
import com.esotericsoftware.kryo.Kryo;

public class NetworkUtil {

    public NetworkUtil() {
    }


    public static void register(Kryo kryo) {
        kryo.register(String.class);
        kryo.register(Double.class);
        kryo.register(SimpleRequest.class);
        kryo.register(SimpleResponse.class);
    }

}

File: src/main/kotlin/NetworkUtil.kt
import com.esotericsoftware.kryo.Kryo

class NetworkUtil {
    companion object {
        internal fun register(kryo: Kryo) {
            kryo.register(String::class.java)
            kryo.register(Double::class.java)
            kryo.register(SimpleRequest::class.java)
            kryo.register(SimpleResponse::class.java)
        }
    }

}
